templates/utils.html

```diff
@@ -156,3 +156,25 @@ <h2 class="post_title">
 	</div>
 </div>
 {%- endmacro %}
+
```

Comment:
@Daniel-Valentine: Be warned that this may a merge conflict with the branch in #637.

Reply:
understood

Reply:
@potatoesAreGod: This was a note to myself -- you don't have to worry about this.